doc/user/content/manage/blue-green.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Blue/Green Deployments"
+description: "How to perform Blue/Green Deployments in Materialize."
+menu:
+  main:
+    parent: manage
+    name: "Blue/Green Deployments"
+    weight: 11
+---
+
+Materialize offers some helpful tools to manage Blue/Green deployments. We recommend using a Blue/Green style workflow any time you need to deploy schema or view definition changes to production views.
+
+## Structuring your environment
+
+{{< note >}}
+We highly recommend using dbt to manage deployments, but this workflow can be recreated using your own custom scripts.
+{{</ note >}}
+
+1. Use schemas and clusters to isolate your changes. Schemas should be the primary container for database objects (views, indexes, materialized views), while clusters are the compute resources that will perform view maintenance and serve queries.
+2. Configure `profiles.yml` to set up the different targets. Use consistent naming across the clusters and schemas that will be deployed together.
+
+  ```yaml
+  default:
+    outputs:
+
+      prod:
+        type: materialize
+        threads: 1
+        host: <host>
+        port: 6875
+        user: <user@domain.com>
+        pass: <password>
+        database: materialize
+        schema: prod
+        cluster: prod
+        sslmode: require
+      prod_deploy:
+        type: materialize
+        threads: 1
+        host: <host>
+        port: 6875
+        user: <user@domain.com>
+        pass: <password>
+        database: materialize
+        schema: prod_deploy
+        cluster: prod_deploy
+        sslmode: require
+
+  target: prod_deploy
+  ```
+
+3. Leave your sources in the public schema and don't touch them. Instead, define them as `sources` in a `schema.yml` file. Since the same source can be shared across your production views and clusters, you won’t need to recreate them.
```

Comment:
Instead of specifying to leave them in the `public` schema, maybe we suggest just a different schema, e.g. "leave your sources in a different schema (e.g. `public`) ..."